enacts/config-sng.yaml

```diff
@@ -30,13 +30,30 @@ shapes_adm:
       is_checked: False
 
 # Daily ENACTS Zarr conversion
-rr_mrg_nc_path: /Data/data23/ANACIM/anacim_v3/ALL_block2/rr_mrg_daily/
-rr_mrg_resolution: .0375
-rr_mrg_chunks:
+nc_path: /Data/data23/ANACIM/anacim_v3/ALL_block2/rr_mrg_daily/
+zarr_resolution: null #set to null for no regridding
+chunks: #consider changing chunk sizes if regridding
     T: 592
     Y: 31
     X: 29
-rr_mrg_zarr_path: /data/aaron/ANACIM/anacim_v3/ALL_block2/rr_mrg_daily
+zarr_path: /data/aaron/ANACIM/anacim_v3/ALL_block2/rr_mrg_daily
```

Comment:
here it's not going to work out: ANACIM/anacim_v3/ALL_block2/ need be either with the paths, or with the vars.